webv2/updateTableSchema/reviewcolumntype.go

```diff
@@ -0,0 +1,212 @@
+package updateTableSchema
```

Comment:
Please rename the file to be in lower camel case. Same for other files.

Reply:
Renamed to snake case, which is the file naming convention of Go for this project